docs/docs/connectors/slack.mdx

```diff
@@ -115,7 +115,8 @@ your bot and tell you about new messages. If you are running locally, you can
 
    As a request URL enter the public url of your bot and append `/webhooks/slack/webhook`, e.g.
    `https://<host>/webhooks/slack/webhook` replacing `<host>` with your URL. If you
-   are using ngrok, your url should look like `https://92832de0.ngrok.io/webhooks/slack/webhook`.
+   are using ngrok, your url should look like `https://92832de0.ngrok.io/webhooks/slack/webhook`. If you're
+   using Rasa X, your url should look like `https://<host>/core/webhooks/slack/webhook`.
```

Comment:
@koaning I believe we removed the need for `core/` in the Rasa X url cc @erohmensing

Reply:
@TyDunn just to check: I did a quick-install using docker compose today and it was still required.

Reply:
I think it will still work with `/core`, but I believe Rick changed it after Ella mentioned that we should just make it consistent, so that you don't have to use it

Reply:
Ah! You're right! I just checked.  In that case, I should change the text.

Reply:
It's now a *very* small change. But I suppose it's better that way. 😅